src/commands/convert/index.ts

```diff
@@ -137,6 +137,13 @@ export default class Convert extends FingerprintingConvertCommand {
       fs.writeFileSync(checkSuffix(flags.output), JSON.stringify(converter.toHdf()))
       break
     }
+
+    default: {
+      throw new Error(`Unknown filetype provided: ${getInputFilename()}\
+        \nThe generic convert command should only be used for taking supported file-based security results and converting into Heimdall Data Format\
+        \nFor more information, run "saf convert --help"\
+        \nFor a list of supported formats, see https://github.com/mitre/heimdall2/tree/master/libs/hdf-converters#supported-formats`)
```

Comment:
Hardcoding a url in like that leads to a host of potential issues.  What happens if we change where hdf-converters is again for some reason.  What if we added support for the file type that they're trying to convert and so it's listed on that page, but they're using an old version of the saf cli that doesn't support that tool (ex. prisma til you realized it was missing)?  Easiest solution: don't say this line at all.  Harder solution: have a `saf convert --supported` or a longer `saf convert --help` that lists out all the mappers supported by the current version of the tool and then a suggestion that new filetypes may be supported so make sure to be using an up to date version of the tool.

Reply:
the sad convert --help is already very long, since it lists the desc for every command. I think I'll just take out the line